dapps/digital-twin-detail/src/components/data/DetailDataGeneral.ts

```diff
@@ -0,0 +1,43 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+// vue imports
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { EvanComponent } from '@evan.network/ui-vue-core';
+import { EvanUIDigitalTwin } from 'dapps/digital-twin.lib';
```

Comment:
relative imports better after absolute imports. I'm also not sure what is the base path in webpack in our build env ;)

Reply:
The paths come from the tsconfig.json. I don't really see the benefit of relative paths here? Webpack and TS will have trouble with those most likely.